libgraphics/src/context.cpp

```diff
@@ -2153,6 +2153,8 @@ static bool MCGContextApplyPaintSettingsToSkPaint(MCGContextRef self, MCGColor p
 			case kMCGImageFilterHigh:
                 r_paint . setFilterLevel(SkPaint::kHigh_FilterLevel);
 				break;
+            default:
+                break;
```

Comment:
@peter-b hmm... ok, moving it to `module-canvas.cpp` which is the only place it's used and defining it as:

```
static const uint8_t kMCGImageFilterCount = 4;
```